models/articles_model.js

```diff
@@ -1,6 +1,19 @@
 const db = require('../db/connection');
 
-exports.selectArticleById = function selectArticleById(article_id) {
+exports.selectAllArticles = () => {
+    return db.query('SELECT articles.*, COUNT(comments.comment_id) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC')
+    .then(({rows}) => {
+        console.log(rows);
+        const newArticleArr = rows.map((articleObj) => {
+            const newArticleObj = {...articleObj};
+            delete newArticleObj.body;
+            return newArticleObj;
+        })
+        return newArticleArr;
+    })
+}
```

Comment:
This is certainly one way of tackling this. I'd prefer to see it all done in one query rather than remove the body after the fact. You might also want to refactor around your comment_count as it is currently a string not a number.